src/pages/components/FooterSection.js

```diff
@@ -0,0 +1,86 @@
+            <p className="leading-[65px] text-center text-5xl text-white font-bold font-nunito">
+              Your <span className="text-yellow-900">bestfriend</span> is <br />
+              waiting for <span className="text-yellow-900">you</span>
+            </p>
+          </div>
+          <div className="flex justify-center">
+            <DownloadButton
+              src="./images/GooglePlay.svg"
+              alt="google play store"
+              upperText="ANDROID APP ON"
+              lowerText="Google Play"
+              link="https://play.google.com/store"
+            />
+            <DownloadButton
+              src="./images/AppStore.svg"
+              alt="apple app store"
+              upperText="DOWNLOAD ON THE"
+              lowerText="App Store"
+              link="https://www.apple.com/ph/app-store/"
+            />
+          </div>
+        </div>
+      </section>
+
+      <section className="flex flex-col px-[20px] sm:px-[20px] md:px-[55px] lg:px-[200px]">
+        <div className="flex flex-col justify-between items-center md:flex-row">
+          <div className="flex flex-col justify-center items-center w-80 pb-6 md:pb-12 mt-7">
+            <img src="/images/LogoDarkFit.svg" alt="LogoDark" />
+            <p className="text-center md:text-left mt-8">
+              Swipet is a mobile application that makes pet adoption fun and
+              easier.
+            </p>
+          </div>
+          <div className="flex py-8 flex-col justify-center items-center font-nunito text-text-header-dark-color md:items-start">
+            <div className="font-bold text-2xl">Links</div>
+            <a href="/FAQs" className="mt-7">
+              FAQs
+            </a>
+            <a href="/about" className="mt-2 mb-2">
+              About
+            </a>
+            <a href="/term-and-conditions">Term and Conditions</a>
+          </div>
+          <div className="flex py-8 flex-col justify-center items-center font-nunito text-text-header-dark-color md:items-start">
+            <div className="font-bold text-2xl ">Contact</div>
+            <div className="mt-7">+63-905-478-9285</div>
+            <div className="mt-2 mb-2">swipet.contact@sun-asterisk.com</div>
+            <div>Don Gil Garcia St, Cebu City</div>
+          </div>
+        </div>
+        <hr />
+        <div className="flex flex-col justify-between items-center gap-10 my-11 md:flex-row">
+          <div className="w-80 flex justify-center gap-12 md:justify-start">
+            <img
+              src="/images/Facebook.svg"
+              className="mediaIconStyle"
+              alt="facebook icon"
+            />
+            <img
+              src="/images/Instagram.svg"
+              className="mediaIconStyle"
+              alt="instagram icon"
+            />
+            <img
+              src="/images/Twitter.svg"
+              alt="twitter icon"
+              className="mediaIconStyle"
+            />
+          </div>
```

Comment:
```suggestion
            <a href="https://www.facebook.com/">
              <img
                src="/images/Facebook.svg"
                className="mediaIconStyle"
                alt="facebook icon"
              />
            </a>
```

These are not only images, it's clickable and has links to it. We can use anchor tags since the page is not part of our application